airbnb_project/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.8'
+services:
+  db:
+    image: postgres:latest
+    container_name: postgres_db
+    env_file:
+      - ./.env
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+      - ./database/airbnb_db_creator.sql:/docker-entrypoint-initdb.d/airbnb_db_creator.sql
```

Comment:
I think there's no need to initialize db with `./database/airbnb_db_creator.sql`, so can be removed in the future. In django, this can be done with `python manage.py makemigrations`, then `python manage.py migrate`